src/components/wizard-navigation-bar.component.horizontal.less

```diff
@@ -29,7 +29,7 @@
 @text-padding-bottom: 10px;
 @text-height: 14px;
 
-:host.horizontal {
+.horizontal {
```

Comment:
shouldn't this also be `wizard-navigation-bar.horizontal`, like for the vertical file?

Reply:
Just changed it as requested